nimbus-consensus/src/import_queue.rs

```diff
@@ -76,24 +76,20 @@ where
 
 		debug!(target: crate::LOG_TARGET, "🪲 Signature according to verifier is {:?}", signature);
 
-		// Grab the digest from the runtime
-		//TODO use the trait. Maybe this code should move to the trait.
-		let consensus_digest = block_params.header
+		// Grab the author information from either the preruntime digest or the consensus digest
+		//TODO use the trait
+		let claimed_author = block_params.header
 			.digest()
 			.logs
 			.iter()
-			.find(|digest| {
+			.find_map(|digest| {
 				match *digest {
-					DigestItem::Consensus(id, _) if id == b"nmbs" => true,
-					_ => false,
+					DigestItem::Consensus(id, ref author_id) if id == NIMBUS_ENGINE_ID => Some(author_id.clone()),
+					DigestItem::PreRuntime(id, ref author_id) if id == NIMBUS_ENGINE_ID => Some(author_id.clone()),
+					_ => None,
```

Comment:
@joelamouche This is the change necessary to extract author information. Previously we just used the Consensus Digest. Now wee will use either the consensus digest or the preruntime digest. Eventually, support for the Consensus digest will be dropped.